### PR TITLE
Change gitpod button to HTML link

### LINKED ISSUE #1246: links: option to prevent external icon (for images e.g. buttons)
To prevent this:

image
> From https://dvc.org/doc/user-guide/contributing/docs#development-environment

### PR DESCRIPTION
**Pull request by rogermparent**
This is a workaround to skip the automatic "external" icon appended to normal links in docs MD.

Fixes? #1246

### CODE REVIEW

File: content/docs/user-guide/contributing/docs.md [hunk 1]
```diff
@@ -63,7 +63,9 @@ formatted and linted as well, which is also ensured by the full setup below.
 > dependencies, run `yarn build`, and start the webserver so that you can start
 > straight away:
 
-[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/iterative/dvc.org)
+<a href="https://gitpod.io/#https://github.com/iterative/dvc.org">
+  <img src="https://gitpod.io/button/open-in-gitpod.svg" alt="Open in Gitpod">
+</a>
```
Comment (jorgeorpinel) [inline content/docs/user-guide/contributing/docs.md:68]: What if we eant to put the button inside the block quote above? It's an actual formatting change in my regular updates PR. Will it work?

Comment (jorgeorpinel) [reply content/docs/user-guide/contributing/docs.md:68]: I.e. something like
```suggestion
> straight away:
>
> <a href="https://gitpod.io/#https://github.com/iterative/dvc.org">
>   <img src="https://gitpod.io/button/open-in-gitpod.svg" alt="Open in Gitpod">
> </a>
```

Comment (rogermparent) [reply content/docs/user-guide/contributing/docs.md]: Surprisingly, this worked with no issue!

Comment (jorgeorpinel) [reply content/docs/user-guide/contributing/docs.md]: I see that. Noice

**Comment by jorgeorpinel** [comment]
Noice. This doesn't close #1246 because Ivan has a question in https://github.com/iterative/dvc.org/issues/1246#issuecomment-624382598. Let's close it (or not) manually.

**Pull request merged**